src/features/posts/components/post-card.tsx

```diff
@@ -0,0 +1,19 @@
+import { PostRequest } from "../../../shared/types/post-request"
```

Comment:
To nie jest PostRequest, jak już to PostResponse albo Post/PostDto

Reply:
Request może być jak robisz z tym obiektem Request np do api. Tutaj go używasz więc używasz raczej obiektu typu Response